src/browser/api/window.js

```diff
@@ -477,7 +477,7 @@ Window.create = function(id, opts) {
             });
 
             // can't unhook when the 'closed' event fires; browserWindow is already destroyed then
-            browserWindow.webContents.removeAllListeners('page-favicon-updated');
+            webContents.removeAllListeners('page-favicon-updated');
```

Comment:
➕ ➕ 

@joneit when you have a sec can you review @rdepena's concern, the 6 and 7 hotfixes rely on this